dbf2csv/demo/conversion_demo.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<odoo>
+
+    <record id="demo_conversion_1" model="dbftocsv.conversion">
+        <field name="name">Demo conversion 1</field>
+        <field name="data_file" type="base64" file="dbf2csv/static/file/demo.dbf"/>
```

Comment:
I guess the file should be in the `demo` directory, not in static...

Reply:
Ok